### PR TITLE
bookmarks shared with anonymous users can be viewed by everyone

### PR DESCRIPTION
**Pull request by philsogaDGI**

### CODE REVIEW

File: islandora_bookmark.module [hunk 1]
```diff
@@ -727,7 +727,7 @@ function islandora_bookmark_detailed_form(array $form, array &$form_state, $list
           $row = $form_state['triggering_element']['#row'];
           $remove_user = $form_state['values']['bookmarks']['users'][$row]['user'];
           if ($remove_user == 0) {
-            $output_user = variable_get('anonymous', t('Anonymous'));
+            $output_user = variable_get('anonymous', t('Everyone'));
```
Comment (jordandukart) [inline islandora_bookmark.module]: The second argument passed to variable_get is the value used is the default for when the `anonymous` variable isn't set. Anonymous is a defined Drupal variable so this `Everyone` value will never get used.

Comment (adam-vessey) [inline islandora_bookmark.module]: Yeah. The default should be consistent with the value in https://api.drupal.org/api/drupal/modules!user!user.admin.inc/function/user_admin_settings/7

File: islandora_bookmark.module [hunk 2]
```diff
@@ -773,7 +773,7 @@ function islandora_bookmark_detailed_form(array $form, array &$form_state, $list
           foreach ($bookmark_users as $itr => $uid) {
             $tempuser = user_load($uid);
             if ($tempuser->uid == 0) {
-              $user_output = variable_get('anonymous', t('Anonymous'));
+              $user_output = variable_get('anonymous', t('Everyone'));
```
Comment (adam-vessey) [inline islandora_bookmark.module]: Should be consistent with the value on https://api.drupal.org/api/drupal/modules!user!user.admin.inc/function/user_admin_settings/7

File: islandora_bookmark.module [hunk 3]
```diff
@@ -816,7 +816,7 @@ function islandora_bookmark_detailed_form(array $form, array &$form_state, $list
         if ($user->uid != 0) {
           $options['0'] = array(
             'uid' => 0,
-            'name' => variable_get('anonymous', 'anonymous'),
+            'name' => variable_get('anonymous', 'Everyone'),
```
Comment (adam-vessey) [inline islandora_bookmark.module]: Should be consistent with the value on https://api.drupal.org/api/drupal/modules!user!user.admin.inc/function/user_admin_settings/7

File: islandora_bookmark.module (hunk 1 above)
Comment (adam-vessey) [inline islandora_bookmark.module]: Or possibly, just get rid of the `variable_get()`s entirely? "Everyone" does kinda make sense now, with how we're using them.

**Comment by adam-vessey** [comment]
Can't actually go to the shared list as another user--gets "Access Denied".  Probably have to actually adjust the access callback of the menu path.

File: islandora_bookmark.module [hunk 4]
```diff
@@ -96,9 +96,8 @@ function islandora_bookmark_access($list_id) {
   // The error that codesniffer was giving was Using the e flag in preg_match.
```
Comment (jonathangreen) [inline islandora_bookmark.module]: Should these comments go as well?

**Comment by adam-vessey** [comment]
Should also get rid of the two other similar code standard ignore blocks which were triggering the same error:
- https://github.com/philsogaDGI/islandora_bookmark/blob/e8cd8a1b53d6ff8a44e82b3b3a55b3201cfd0a52/islandora_bookmark.module#L143-L148
- https://github.com/philsogaDGI/islandora_bookmark/blob/e8cd8a1b53d6ff8a44e82b3b3a55b3201cfd0a52/includes/api.inc#L134-L139

**Comment by jordandukart** [comment]
There are inconsistencies of how we are handling the replacement of "anonymous" to "Everyone". There still exists three occurances where we are refering to "anonymous". 

https://github.com/philsogaDGI/islandora_bookmark/blob/1ee369c0b4fc8f4c4a475a5231b5cb9cccbf9759/islandora_bookmark.module#L699-L701
https://github.com/philsogaDGI/islandora_bookmark/blob/1ee369c0b4fc8f4c4a475a5231b5cb9cccbf9759/islandora_bookmark.module#L824-L826
https://github.com/philsogaDGI/islandora_bookmark/blob/1ee369c0b4fc8f4c4a475a5231b5cb9cccbf9759/islandora_bookmark.module#L1423-L1425

**Pull request merged**